tests/test_persistence.py

```diff
@@ -64,40 +65,39 @@ def test_init_dryrun_db(default_conf, tmpdir):
 
 
 @pytest.mark.usefixtures("init_persistence")
-def test_update_with_binance(limit_buy_order, limit_sell_order, fee, caplog):
+def test_update_limit_order(limit_buy_order_usdt, limit_sell_order_usdt, fee, caplog):
     """
-    On this test we will buy and sell a crypto currency.
-
-    Buy
-    - Buy: 90.99181073 Crypto at 0.00001099 BTC
-        (90.99181073*0.00001099 = 0.0009999 BTC)
-    - Buying fee: 0.25%
-    - Total cost of buy trade: 0.001002500 BTC
-        ((90.99181073*0.00001099) + ((90.99181073*0.00001099)*0.0025))
-
-    Sell
-    - Sell: 90.99181073 Crypto at 0.00001173 BTC
-        (90.99181073*0.00001173 = 0,00106733394 BTC)
-    - Selling fee: 0.25%
-    - Total cost of sell trade: 0.001064666 BTC
-        ((90.99181073*0.00001173) - ((90.99181073*0.00001173)*0.0025))
-
-    Profit/Loss: +0.000062166 BTC
-        (Sell:0.001064666 - Buy:0.001002500)
-    Profit/Loss percentage: 0.0620
-        ((0.001064666/0.001002500)-1 = 6.20%)
-
-    :param limit_buy_order:
-    :param limit_sell_order:
-    :return:
+        On this test we will buy and sell a crypto currency.
+        fee: 0.25% quote
+        open_rate: 2.00 quote
+        close_rate: 2.20 quote
+        amount: = 30.0 crypto
+        stake_amount
+            1x,-1x: 60.0  quote
+        borrowed
+             1x:  0 quote
+        open_value: (amount * open_rate) ± (amount * open_rate * fee)
+             1x, 3x: 30 * 2 + 30 * 2 * 0.0025 = 60.15 quote
```

Comment:
Same than in the other comment - be careful with mentioning "leverage" in PR's that are intended to be merged to develop.
While the comment may be correct in the future - it's incorrect if we look at this PR alone - and incorrect in the state it's going to result in develop.